oem/firstvoices/ios/FirstVoices/AppDelegate.swift

```diff
@@ -31,6 +31,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 
     // Replace with your application group id
     Manager.applicationGroupIdentifier = FVConstants.groupID
+    Manager.shared.spacebarText = SpacebarText.KEYBOARD
```

Comment:
```
/Users/jahorton/keymanapp/keyman/oem/firstvoices/ios/FirstVoices/AppDelegate.swift:34:35: error: cannot find 'SpacebarText' in scope
    Manager.shared.spacebarText = SpacebarText.KEYBOARD
```

Pretty sure that's because it's declared within `Manager`; testing this now.

Reply:
This was building on my machine. What's going on with the KeymanEngine included in FV Keyboards? Are we getting an outdated version perhaps?